### ISSUE TITLE
Alias WORKDIR to CD

### ISSUE THREAD

**Issue by josegonzalez** [hverr/lxdfile#4]
These two seem analogous to each other, based on my reading of the code/some lxdfiles. Are they the same, or is there a case where you'd think CD doesn't apply to WORKDIR?

**Comment by hverr** [comment]
I’m sorry I apparently missed this issue.

WORKDIR and CD _should_ not be equivalent (I emphasise should because I can’t remember whether they have different semantics in the code).

CD is ephemeral and only remembered during the build process. WORKDIR is persistent. If WORKDIR is used, the container should start in this directory.

**Comment by josegonzalez** [comment]
Ah so the last WORKDIR should be PWD for that image? Otherwise, they should be the ~same, right?

**Comment by hverr** [comment]
Exactly, emphasis on should. At least, I think so, but to be honest I should recheck the Docker documentation.

**Comment by josegonzalez** [comment]
Doesn't look like Dockerfile has a `CD` command, so adding a change in the PWD for the last `WORKDIR` should be enough, as they appear equivalent otherwise.
